### DISCUSSION TITLE
Having instructions for installing Tribler on Fedora?

### DISCUSSION THREAD

**Discussion by Solomon1732** [Tribler/tribler#6015] [General] [answered]
Can instructions for installing Tribler on Fedora be added to https://github.com/Tribler/tribler/blob/main/doc/development/development_on_linux.rst ? Currently it holds instructions only for Debian and Debian-based distros.

**Comment by da2x** [comment, marked as answer]
Use the Flatpak! You can find it [on Flathub](https://flathub.org/apps/details/org.tribler.Tribler).

**Reply by Solomon1732** [reply]
Ah, didn't know it has a Flatpak now. Thanks!
